tests/cli/test_rasa_train.py

```diff
@@ -541,7 +541,7 @@ def test_train_validation_fail_on_warnings(
     request: pytest.FixtureRequest,
 ):
     test_data_dir = Path(request.config.rootdir, "data", "test_moodbot", "data")
-    test_domain = Path(request.config.rootdir, "data", "test_domains", "domain.yml")
+    test_domain = Path(request.config.rootdir, "data", "test_domains", "default.yml")
```

Comment:
Could you please reset to `domain.yml` here too?

Reply:
That was typo in the first place there is no `domain.yaml` in that directory.... (and I was the one who wrote that test sadly)